web_api/core/models.py

```diff
@@ -25,6 +34,76 @@ class Meta:
     def is_authenticated(self) -> bool:
         return True
 
+    def sync_accounts(self) -> None:
+        """
+
+        - create any missing installations
+        - add memberships of user for installations
+        - remove memberships of installations that aren't included
+        """
+        user_installations_res = requests.get(
```

Comment:
does requests have a default timeout?

Reply:
no, I've configured the timeout parameter, but I don't think it actually sets a hard timeout, just a timeout on how long to wait for more of a response. I think setting a timeout in gunicorn will be the best way to do this.

https://docs.gunicorn.org/en/stable/settings.html#timeout